models/taskRequests.js

```diff
@@ -476,6 +486,33 @@ const addNewFields = async () => {
   return { documentsModified, totalDocuments };
 };
 
+const addUsersCountAndCreatedAt = async () => {
+  const taskRequestsSnapshots = (await taskRequestsCollection.get()).docs;
+
+  const bulkWriter = firestore.bulkWriter();
+  let documentsModified = 0;
+  const totalDocuments = taskRequestsSnapshots.length;
+  taskRequestsSnapshots.forEach((taskRequestsSnapshot) => {
+    const taskRequestData = taskRequestsSnapshot.data();
+    let isDocumentModified = false;
+    if (!taskRequestData.usersCount) {
+      taskRequestData.usersCount = taskRequestData.users.length;
+      isDocumentModified = true;
+    }
+    if (!taskRequestData.createdAt) {
+      taskRequestData.createdAt = 0;
```

Comment:
assigning it 0 will mean we are making the date `Thu Jan 01 1970` 👀

Reply:
- Yes, since we do not know when it was created. 
- The field is needed so that firestore will include these documents when we sort.

Reply:
we can use some backfill date such as `Jan 1st 2020` as default date or some similar date when RDS was found. During a similar conversation between Ritik and Ankush, Ankush suggested using a similar approach of using a date like "Jan 1st, 2020" as a backfill date. You can confirm the same in morning standup.

Reply:
But this is not a blocker hence I am approving this PR.

Reply:
`QueryDocumentSnapshot` contains the document created time. So using that here.